src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,10 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+
+    private int weight;
+    private int maxScoopWeight;
```

Comment:
how can i get access to these fields?

Reply:
I've added getters and setters for these fields